src/applications/vaos/new-appointment/redux/helpers/formSubmitTransformers.v2.js

```diff
@@ -40,13 +40,17 @@ export function transformFormToVAOSCCRequest(state) {
     residentialAddress.addressType !== 'MILITARY OVERSEAS'
   ) {
     preferredLocation = {
+      line: parentFacility.address.line,
       city: residentialAddress.city,
       state: residentialAddress.stateCode,
+      postalCode: parentFacility.address.postalCode,
```

Comment:
In the transformFormToVAOSCCRequest function, I think we can remove the check for  `residentialAddress` since we took away the isResidentalAddress criteria a while back. Meaning residential address could be null and veteran can still submit a request as long as CC eligible.

preferredLocation refers to the selected provider's location. 
```suggestion
    preferredLocation = {
      line: provider.address.line[0],
      city: provider.address.city,
      state: provider.address.state,
      postalCode: provider.address.postalCode,
```
Will somehow need to concat address line array if more than one address line exist.

Reply:
The VAOS Service swagger shows to have preferredLocation with only two attributes, `city` and `state`. The Middleware team needs to add `line` and `postalCode` for us to send the complete provider address in the POST new appointment request.

Reply:
Ok. Turns out this was all wrong. So I'm changing everything back to what it originally was. The address information should be under 'practitioners'. Brad is updating the Swagger to reflect this. I also asked if 'preferredLocation' should be removed since it appears that it won't be used anymore. It was suggested to delete it later.